go.sum

```diff
@@ -339,6 +339,7 @@ github.com/golang/groupcache v0.0.0-20190129154638-5b532d6fd5ef h1:veQD95Isof8w9
 github.com/golang/groupcache v0.0.0-20190129154638-5b532d6fd5ef/go.mod h1:cIg4eruTrX1D+g88fzRXU5OdNfaM+9IcxsU14FzY7Hc=
 github.com/golang/groupcache v0.0.0-20190702054246-869f871628b6 h1:ZgQEtGgCBiWRM39fZuwSd1LwSqqSW0hOdXCYYDX0R3I=
 github.com/golang/groupcache v0.0.0-20190702054246-869f871628b6/go.mod h1:cIg4eruTrX1D+g88fzRXU5OdNfaM+9IcxsU14FzY7Hc=
+github.com/golang/groupcache v0.0.0-20191227052852-215e87163ea7 h1:5ZkaAPbicIKTF2I64qf5Fh8Aa83Q/dnOafMYV0OMwjA=
```

Comment:
Why is there a change in the sum file?

Reply:
Go auto-updated that dependency and I can't figure out why. I no longer understand go modules.

Reply:
Okay, so in tracing things, it appears that this came from a rebase, not from Go itself. I'll try removing it.

Reply:
I manually removed the line. But it looks like there was a problem in the go modules file (notice a missing entry) that the go module tool chain tried to fix during a rebase. It seems that the local compile will work without this line. But my guess is that the next time somebody updates or rebuilds the module cache, this exact same thing will happen again. That said... this PR can do without it, and I am tired of trying to deal with Go modules. So removing it is fine for this PR.